packages/babel-plugin-transform-class-static-block/test/plugin-ordering.test.js

```diff
@@ -17,17 +18,18 @@ describe("plugin ordering", () => {
         highlightCode: false,
         configFile: false,
         babelrc: false,
-        plugins: [proposalClassProperties, proposalClassStaticBlock],
+        plugins: [
+          proposalClassProperties,
+          proposalClassStaticBlock,
+          externalHelpers,
+        ],
       }).code,
     ).toMatchInlineSnapshot(`
       "var _Foo;
-      function _defineProperty(obj, key, value) { key = _toPropertyKey(key); if (key in obj) { Object.defineProperty(obj, key, { value: value, enumerable: true, configurable: true, writable: true }); } else { obj[key] = value; } return obj; }
-      function _toPropertyKey(t) { var i = _toPrimitive(t, \\"string\\"); return \\"symbol\\" == typeof i ? i : i + \\"\\"; }
-      function _toPrimitive(t, r) { if (\\"object\\" != typeof t || !t) return t; var e = t[Symbol.toPrimitive]; if (void 0 !== e) { var i = e.call(t, r || \\"default\\"); if (\\"object\\" != typeof i) return i; throw new TypeError(\\"@@toPrimitive must return a primitive value.\\"); } return (\\"string\\" === r ? String : Number)(t); }
```

Comment:
When mixing Babel 7 and Babel 8, `toPropertyKey` uses `String(i)` instead of `i + ""`. Since that's not the point of this test, I just updated it to use external helpers

Reply:
This could mean we are using `@babel/helpers@8.0.0-alpha.7`, is this expected?

Reply:
Yes, because that's the version used by `@babel/core@8` (when testing Babel 7 plugins with Babel core 8).